meta/pattern-template.md

```diff
@@ -1,32 +1,38 @@
 ## Title  
 
 ## Problem
-What is the problem - crisp definition of the problem
+What is the problem - crisp definition of the problem. 1-2 sentances. Anymore should go into other sections.
 
```

Comment:
## Story (Optional)
Sometimes there is a short story that succinctly illustrates the problem, giving useful background information.